core/src/main/java/org/elasticsearch/rest/action/support/RestActions.java

```diff
@@ -64,18 +73,24 @@ public static long parseVersion(RestRequest request, long defaultVersion) {
     }
 
     static final class Fields {
+        static final String _NODES = "_nodes";
```

Comment:
Yeah, but you still get the wasted, additional reference to the constant pool (granted that's small potatoes, it's something that the compiler could auto-replace for us). Agreed!